build_scripts/ci_script.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 """A simple script to run CI build steps."""
 
+
```

Comment:
revert all the changes that merely add a newline to a file, in the interests of not touching unnecessary files in an already large change